subsys/bluetooth/controller/ll_sw/ull_conn_internal.h

```diff
@@ -7,17 +7,52 @@
 /* Macro to convert time in us to connection interval units */
 #define RADIO_CONN_EVENTS(x, y) ((u16_t)(((x) + (y) - 1) / (y)))
 
-/* Macro to return PDU time */
+/*
+ * Macros to return correct PDU time
+ * Note: formula is valid for S=8 coding only
+ * see BT spec Version 5.1 Vol 6. Part B, chapters 2.1 and 2.2
+ * for packet formats and thus lengths
+ *
+ * Payload overhead size is the BT address + the MIC
+ */
+#define PAYLOAD_OVERHEAD_SIZE (2+4)
+
 #if defined(CONFIG_BT_CTLR_PHY_CODED)
-#define PKT_US(octets, phy) \
-	(((phy) & BIT(2)) ? \
-	 (80 + 256 + 16 + 24 + ((((2 + (octets) + 4) * 8) + 24 + 3) * 8)) : \
-	 (((octets) + 14) * 8 / BIT(((phy) & 0x03) >> 1)))
+#define CODED_PHY_PREAMBLE_TIME_US (80)
+#define CODED_PHY_ACCESS_ADDRESS_TIME_US (256)
+#define CODED_PHY_CI_TIME_US (16)
+#define CODED_PHY_TERM1_TIME_US (24)
+#define CODED_PHY_CRC_SIZE (24)
+#define CODED_PHY_TERM2_SIZE (3)
+
+#define FEC_BLOCK1_TIME_US (CODED_PHY_ACCESS_ADDRESS_TIME_US + \
```

Comment:
Just thinking aloud, left hand side is more readable, relate-able to the figure in BT Spec. and less time spent in implementing!
Otherwise changes look good.